.pre-commit-config.yaml

```diff
@@ -25,10 +25,14 @@ repos:
       - id: poetry-lock
       - id: poetry-install
       - id: pre-commit-install
-      - id: megalinter
-        args: &megalinter-args [--flavor, documentation, --release, v5.16.1]
```

Comment:
The commit message has the wrong "from" version.

Reply:
Oh no! Thanks for catching this!